src/effects/VST3/internal/x11/RunLoop.h

```diff
@@ -0,0 +1,66 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file RunLoop.h
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
+
+**********************************************************************/
+
+#pragma once
+
+#include <unordered_map>
+#include <gtk/gtk.h>
+#include <gdk/gdkx.h>
+#include <gdk/gdk.h>
+
+#include <pluginterfaces/gui/iplugview.h>
+
+namespace internal
+{
+    
+namespace x11
+{
+
+class RunLoop : public Steinberg::Linux::IRunLoop
+{
+   struct EventData
+   {
```

Comment:
I suggested a constructor, move constructor, and destructor in this structure for simpler resource management